static/app/components/charts/miniBarChart.tsx

```diff
@@ -141,7 +141,7 @@ interface Props extends Omit<BaseChartProps, 'css' | 'colors' | 'series' | 'heig
   /**
    * Colors to use on the chart.
    */
-  colors?: string[];
+  colors?: ReadonlyArray<string>;
```

Comment:
We should probably talk about `ReadonlyArray` and the `readonly` keyword at TSC. It's a tool folks could use more, and with noUncheckedIndexedAccess there's motivation.

Reply:
Great idea!